src/acl_hostch.cpp

```diff
@@ -1015,19 +1033,18 @@ void acl_write_program_hostpipe(void *user_data, acl_device_op_t *op) {
         host_pipe_info.m_physical_device_id, data_reg,
         event->cmd.info.host_pipe_dynamic_info.write_ptr,
         event->cmd.info.host_pipe_dynamic_info.size);
+
     if (status != 0) {
       acl_mutex_unlock(&(host_pipe_info.m_lock));
       acl_set_device_op_execution_status(op, -1);
       return;
     }
 
-    // For now, we trust the AVALON_MM by default uses valid.
-    // TODO: fix this later by using the new protocol info
-    // provided by the compiler.
```

Comment:
protocol is not being used at this moment. Instead of adding new protocol value, we decided to use another field to encapsulate this kind of information. protocol value might be used in the future, e.g. if Runtime needs to differentiate between AVALON_Streaming and AXI streaming, or when another new protocol is supported.

Reply:
Okay sounds good! So that other people who looks at this won't get confused, would you mind adding a comment in acl_auto_configure.cpp where this value is parsed to say this value is not currently used and is reserved for the future?

Reply:
Sounds good! Added the comment